support-frontend/assets/pages/new-contributions-landing/components/ContributionAmount.jsx

```diff
@@ -97,6 +108,82 @@ const iconForCountryGroup = (countryGroupId: CountryGroupId): React$Element<*> =
   }
 };
 
+const localisedAverageAmountSentence: {
+  [CountryGroupId]: {
+    amountSentence: string,
+    averageAnnualAmount: number,
+  }
+} = {
+  GBPCountries: {
+    amountSentence: 'In the UK, the',
+    averageAnnualAmount: 50.37,
+  },
+  UnitedStates: {
+    amountSentence: 'In the US, the',
+    averageAnnualAmount: 50.16,
+  },
+  AUDCountries: {
+    amountSentence: 'In Australia, the',
+    averageAnnualAmount: 82.65,
+  },
+  EURCountries: {
+    amountSentence: 'In Europe, the',
+    averageAnnualAmount: 52.92,
+  },
+  NZDCountries: {
+    amountSentence: 'In New Zealand, the',
+    averageAnnualAmount: 58.51,
+  },
+  Canada: {
+    amountSentence: 'In Canada, the',
+    averageAnnualAmount: 62.68,
+  },
+  International: {
+    amountSentence: 'The',
+    averageAnnualAmount:  61.78,
+  },
+};
+
+
+function averageAmountVariantCopy(
+  countryGroupId: CountryGroupId,
+  contributionType: ContributionType,
+  currencyString: string,
+) {
+  const localAverageAmountInfo = localisedAverageAmountSentence[countryGroupId];
+  const localAverageAmount = Math.round(localAverageAmountInfo.averageAnnualAmount);
+  const localAverageAmountSentence = localAverageAmountInfo.amountSentence;
+  return `Please select the amount you'd like to contribute each year. ${localAverageAmountSentence} average choice is ${currencyString}${localAverageAmount}.`;
```

Comment:
this is the mean, and for a mean you can only do that on a quantity.  For an average choice, it would have to be the mode (ie most popular choice).
Probably this would only seem confusing to a maths/science type person, but for me it would feel more trustworthy if it was more clear what the number represented.

Reply:
Good point. We are going to remove the world 'choice'